conanfile.py

```diff
@@ -40,7 +40,7 @@ def requirements(self):
         self.tool_requires("cmake/[>=3.19]")
         self.requires("fmilibrary/[~2.3]")
         self.requires("libzip/[>=1.7 <1.10]") # 1.10 deprecates some functions we use
-        self.requires("ms-gsl/[>=3 <5]", transitive_headers=True)
+        self.requires("ms-gsl/[>=3 <5]", transitive_headers=True, transitive_libs=True)
```

Comment:
Why is this necessary? As far as I know, MS GSL is a header-only library.

Reply:
I also think it should be okay with header only transitivity. But we had an issue in cmake finding the library without the option sometime ago. I will try to remove it and see if it is still causing the problem

Reply:
Seems okay after removal.